make/test.mk

```diff
@@ -94,8 +94,8 @@ endif
 .PHONY: test-e2e-olm-ci
 test-e2e-olm-ci: get-test-namespace ./vendor
 	$(Q)oc new-project $(TEST_NAMESPACE)
-	$(Q)sed -e "s,REPLACE_IMAGE,registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:devconsole-operator-registry," ./test/e2e/catalog_source_OS4.yaml | oc apply -f -
-	$(Q)oc apply -f ./test/e2e/subscription_OS4.yaml
+	$(Q)sed -e "s,REPLACE_IMAGE,registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:devconsole-operator-registry," ./test/e2e/catalog_source_OS4.yaml | oc apply -f - --namespace $(TEST_NAMESPACE)
```

Comment:
This should ideally suffice https://github.com/redhat-developer/devconsole-operator/blob/master/test/e2e/catalog_source_OS4.yaml#L6

Reply:
That's correct. I am closing this PR.